test/e2e_node/e2e_node_suite_test.go

```diff
@@ -39,8 +49,44 @@ func TestE2eNode(t *testing.T) {
 
 // Setup the kubelet on the node
 var _ = BeforeSuite(func() {
+	if *buildServices {
+		BuildGo()
+	}
+	if *nodeName == "" {
+		output, err := exec.Command("hostname").CombinedOutput()
+		if err != nil {
+			glog.Fatal("Could not get node name from hostname %v.  Output:\n%s", err, output)
+		}
+		*nodeName = strings.TrimSpace(fmt.Sprintf("%s", output))
+	}
+
+	if *startServices {
+		e2eService = NewE2eService()
+		if err := e2eService.Start(); err != nil {
+			Fail(fmt.Sprintf("Unable to start node services.\n%v", err))
+		}
+		glog.Infof("Node services started.  Running tests...")
+	} else {
+		glog.Infof("Running tests without starting services.")
+	}
 })
 
 // Tear down the kubelet on the node
 var _ = AfterSuite(func() {
+	if CurrentGinkgoTestDescription().Failed {
+		glog.Infof("PWITTROC - FAILED")
+	} else {
+		glog.Infof("PWITTROC - PASSED")
+	}
+	if e2eService != nil && *startServices && *stopServices {
+		glog.Infof("Stopping node services...")
+		e2eService.Stop()
+		glog.V(2).Infof("-------------------------------------------------------------")
```

Comment:
(nit) Each info line will have a timestamp & serial number associated with it - maybe it would be cleaner to just build the string and log the whole block at once?